### PR TITLE
added dynamic footer

### PR DESCRIPTION
**Pull request by rajivharlalka**
I have added a dynamic footer which renders the current year dynamically and a message "Made with React Js " which can be changed upto choice

### CODE REVIEW

**Comment by rajivharlalka** [comment]
image

**Comment by Namanbhatia7** [comment]
@rajivharlalka Ah just little tweaks, made with football-data.org(hyperlink this) API and in next line,
copyright {symbol} year...like this

**Comment by rajivharlalka** [comment]
Screenshot (58)

**Comment by rajivharlalka** [comment]
Screenshot (59)

**Comment by Namanbhatia7** [comment]
resolve merge conflicts and you are good to go. Great work!

File: app.js [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
-app.use(express.static(__dirname + '/public'));
+app.set("view engine", "ejs");
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(express.static(__dirname + "/public"));
 app.use(gameRoutes);
 
-app.listen(3000, function(){
-    console.log("Server is Running on Port 3000");
-
+app.listen(4000, function () {
```
Comment (Namanbhatia7) [inline app.js:16]: Did you change Port because of react using 3000 Port?

**Comment by rajivharlalka** [comment]
Oh yeah, actually i was running another React project which was running on
port 3000 ...

File: app.js (hunk 1 above)
Comment (rajivharlalka) [reply app.js:16]: I am again updating this and making a commit ....

File: app.js [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
-app.set('view engine', 'ejs');
-app.use(bodyParser.urlencoded({extended: true}));
-app.use(express.static(__dirname + '/public'));
+app.set("view engine", "ejs");
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(express.static(__dirname + "/public"));
 app.use(gameRoutes);
 
-app.listen(3000, function(){
-    console.log("Server is Running on Port 3000");
```
Comment (Namanbhatia7) [inline app.js:17]: change port in console.log message

**Review by Namanbhatia7 — CHANGES_REQUESTED** [review]

File: app.js (hunk 1 above)
Comment (Namanbhatia7) [reply app.js:16]: No issues, we will have to change that in future, just write new port number in console.log message

File: app.js (hunk 2 above)
Comment (rajivharlalka) [reply app.js:17]: I have updated the port in ap.js

Comment (rajivharlalka) [reply app.js:17]: I am making a final commit again changing both the port and message to 4000

Comment (rajivharlalka) [reply app.js:17]: Updated

**Review by Namanbhatia7 — APPROVED** [review]
Looks good, merging!

**Pull request merged**